multi-agent/tests/unit/blueprints/test_blueprint_document_model.py

```
@@ -0,0 +1,118 @@
+    def test_none_version_replaced_by_default(self):
+        """If a legacy document stored version=None, the default applies."""
+        # Pydantic v2 with default=1 will raise or coerce depending on strictness;
+        # this test documents what actually happens rather than prescribing it.
+        try:
+            doc = BlueprintDocument.model_validate(_minimal_doc_dict(version=None))
+            # If parsing succeeded, version must be valid (≥1)
+            assert doc.version >= 1
+        except Exception:
+            # If Pydantic rejects None, that's acceptable — None is not a
+            # legitimate version in the domain.
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)